Makefile

```diff
@@ -14,7 +14,7 @@ export TOOLS_NAMESPACE=$(OS_NAMESPACE_PREFIX)-tools
 export DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:${POSTGRES_PORT}/${POSTGRES_DB}
 
 # Feature flags
-export FEATURE_PHASE_ALLOCATION=true
+export FEATURE_PHASE_ALLOCATION=false
```

Comment:
Have we have a discussion on turning this off on dev environments? We'll need to make sure we flip the configmaps if that's the case

But I'm not convinced we want to turn this off right away